benchmark_scripts/bm_ind.slurm

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#SBATCH --time=24:00:00
+#SBATCH --nodes=1
```

Comment:
We want to be able to use multiple nodes. Best solution may be to specify number of tasks and tasks-per-node on the command line in the calling script.